content/millennium/r4/security/provenance.md

```diff
@@ -0,0 +1,162 @@
+* [Agent role (source or informant)](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.role){:target="_blank"}
+* [Agent who (Practitioner, Device, or Organization)](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.who){:target="_blank"}
+* [Agent on behalf of](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent.onBehalfOf){:target="_blank"}
+* [Entity role](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.entity.role){:target="_blank"}
+* [Entity what (Continuity of Care DocumentReference id)](https://hl7.org/fhir/R4/provenance-definitions.html#Provenance.entity.what){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:provenance, :r4) %>
+
+## Search
+
+Search for Provenances that meet supplied query parameters:
+
+    GET /Provenance?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required? | Type          | Description
+--------------|-----------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------
+ `_id`        | This or `target` | [`token`]     | The logical resource id associated with the resource. Example: `doc-7845`
+ `target`     | This or `_id` | [`reference`] | Target Reference(s) (usually version specific). Currently we only support AllergyIntolerance, Condition, DocumentReference, Encounter, Immunization, MedicationRequest, Observation and Procedure Reference(s). Example: `DocumentReference/66547`
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Provenance?_id=doc-881057
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_provenance_bundle) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Retrieve by id
+
+List an individual Provenance by its id:
+
+    GET /Provenance/:id
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Provenance/doc-881057
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_provenance_entry) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Create
+
+Create a new Provenance.
+
+    POST /Provenance
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+* Modifier fields should not be provided, and will cause the transaction to fail.
+* To create Provenance for Observation Measurement Reference, will need `Observation.read` scope along with `Provenance.write` scope.
```

Comment:
The ... and ... scopes must be provided in order to create a Provenance with a Observation Measurement target.

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/pull/371/commits/e87e0be1558e277ac583722d167d60828e55216a#diff-f5afbe5fd0ebfe7f5156f94c1d26876961667b00ce5771a4660569b87d1a661fR109).